rules/mocha.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+  rules: {
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-skipped-tests': 'off',
+    'mocha/no-pending-tests': 'error',
```

Comment:
I think this is actually different in (WIP) PRs vs. the `master` branch. On `master` you obviously don't want pending tests while I'm not sure if it's a good think to have a failing build when there's a pending test in the (WIP) PR.